test/TensorFlow/integration.swift

```diff
@@ -228,6 +228,65 @@ public func test_bool_param2(cond: Bool, // expected-warning {{'cond' implicitly
 // CHECK-NEXT: [[PROGRAM:%.*]] = apply [[STARTFN:%.*]](
 // CHECK: cond_br [[BOOLVAL]],
 
+public func test_multiple_ifs(status: Bool // expected-warning {{'status' implicitly copied to the accelerator}}
+) {
+  var a = Tensor<Int32>(0);
+  let b = a;
+  if (status) { // expected-note {{value used here}}
+    a += b;
+  }
+  a += b;
```

Comment:
do we need this statement in the reproducer?

Reply:
Yes, this is needed. Otherwise, I cannot reproduce the bug.